Manifest.toml

```diff
@@ -33,11 +33,11 @@ uuid = "2a0f44e3-6c83-55bd-87e4-b1978d98bd5f"
 
 [[deps.BinaryBuilderBase]]
 deps = ["CodecZlib", "Downloads", "InteractiveUtils", "JSON", "LibGit2", "LibGit2_jll", "Libdl", "Logging", "OrderedCollections", "OutputCollectors", "Pkg", "ProgressMeter", "Random", "SHA", "Scratch", "SimpleBufferStream", "TOML", "Tar", "UUIDs", "p7zip_jll", "pigz_jll"]
-git-tree-sha1 = "9b1c2e9866fdf353f925693d69aa440dec5b4c60"
-repo-rev = "master"
```

Comment:
Keeping this to `master` makes updates easier

Reply:
Gonna be away from my laptop for most of today, if you want to fix that